app/controllers/dashboard_controller.rb

```diff
@@ -1,8 +1,7 @@
 class DashboardController < ApplicationController
  
   def index
-    
     @bookings = policy_scope(Booking).order(created_at: :desc)
-     
+    @favourite_hikes =  @current_user.favorited_by_type('Hike')
```

Comment:
```suggestion
    @favourite_hikes = @current_user.favorited_by_type('Hike')
```
remove duplicated space for code style